src/cdn.ts

```diff
@@ -0,0 +1,6 @@
+import * as ununificlient from '.';
```

Comment:
Anyway, ununifi client would be better to be "default-export" like cosmosclient.
Not high priority

Reply:
fixed